application/src/main/java/org/wecancodeit/serverside/controller/TopicController.java

```diff
@@ -1,4 +1,23 @@
 package org.wecancodeit.serverside.controller;
 
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.wecancodeit.serverside.model.Topics;
+import org.wecancodeit.serverside.repository.TopicRepo;
+
+import javax.annotation.Resource;
+import java.util.Collection;
+import java.util.Optional;
+
 public class TopicController {
+
+    @Resource
+    private TopicRepo topicRepository;
+
+    @GetMapping("/topics")
+    public Collection<Topics> getTopics(){return (Collection<Topics>) topicRepository.findAll();}
+
+    @GetMapping("/topic")
```

Comment:
If we're going to reference an @PathVariable of topicName, that needs to be in the getMapping as "/topic/{topicName}"